Feature/add pre commit (#137)
sivchari/govalid: README.md

[📊 View Complete Benchmarks](test/benchmark/README.md)

### 🔧 Development Setup

For contributors, install lefthook to enable pre-commit checks:

```bash
make install-lefthook
```

> This will install and configure lefthook to run automatic checks before each commit, ensuring code quality and consistency.

Reviewer: ```suggestion
Because of this, lefthook is installed, then the code-base would be checked automatically before each commit, ensuring code quality and consistency.
```
